src/app/login/page.tsx

```diff
@@ -48,12 +51,33 @@ export default function App() {
           height={65}
           style={{ marginTop: '30px', marginLeft: '30px' }}
         />
+
         <LoginBox>
           <LoginContent>
             <WelcomeSign>Welcome</WelcomeSign>
-            <LoginForm changeUserName={setEmail} changePassword={setPassword} />
+
+            <LoginForm
+              isError={isError}
+              changeUserName={setEmail}
+              changePassword={setPassword}
+              showPassword={showPassword}
+            />
+            {showPassword ? (
+              <EyeIcon
+                onClick={() => setShowPassword(false)}
+                style={{ cursor: 'pointer' }}
+              />
+            ) : (
+              <EyeOffIcon
+                onClick={() => setShowPassword(true)}
+                style={{ cursor: 'pointer' }}
+              />
+            )}
+
             {errorMessage && <ErrorMessage>{errorMessage}</ErrorMessage>}
-            <Button onClick={handleLogin}>Log In</Button>
+            <Button style={{ cursor: 'pointer' }} onClick={handleLogin}>
+              Log In
```

Comment:
since this is text that will appear on the front end, please make sure to use the  global text file and match it to a proper text